build/images/ovs/Dockerfile.ubi

```diff
@@ -12,54 +12,51 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# OVS build scripts are only applicable for RHEL 7.x:
-# https://docs.openvswitch.org/en/latest/intro/install/fedora/
-FROM centos:centos7 as ovs-rpms
-
+FROM quay.io/centos/centos:stream9 as ovs-rpms
 # Some patches may not apply cleanly if a non-default version is provided.
 # See build/images/deps/ovs-version for the default version.
 ARG OVS_VERSION
 
 # Install RPM tools and generic build dependencies.
-RUN yum update -y && yum install wget git yum-utils python38 rpm-build epel-release -y
+RUN yum clean all -y && yum -y install wget git yum-utils python3 rpm-build && \
+    sed -i 's/enabled=.*/enabled=1/g' /etc/yum.repos.d/centos.repo
 
 COPY apply-patches.sh /
+COPY charon-logging.conf /tmp
```

Comment:
Is this a new file? I didn't see it in existing ovs folder or the folder in this PR.

Reply:
No. It was not a new file. https://github.com/antrea-io/antrea/blob/a9300b4311503b14f20417fb6f7bbacd78e458a2/build/images/ovs/charon-logging.conf
Previously it was copied in the final stage. I moved this to the build stage and used bind mount in the final image to reduce the intermediate layers introduced by `COPY`.